pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -258,6 +258,8 @@
     // Name of load manager to use
     @FieldContext(dynamic = true)
     private String loadManagerClassName = "com.yahoo.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl";
+    @FieldContext(dynamic = true)
+    private boolean preferLaterVersions = true;
```

Comment:
Can we add the `loadManager` prefix to the variable name, to provide some kind of config scoping?